src/ctelnet.cpp

```diff
@@ -982,8 +985,9 @@ void cTelnet::processTelnetCommand(const string& command)
                 version = version.section(' ', 0, 0);
 
                 int newVersion = version.toInt();
+                QString _smsg;
                 if (mpHost->mServerGUI_Package_version != newVersion) {
-                    QString _smsg = tr("<The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'>")
+                    _smsg = tr("[ INFO ] - The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'.")
                                             .arg(QString::number(newVersion), QString::number(mpHost->mServerGUI_Package_version));
                     mpHost->mpConsole->print(_smsg.toLatin1().data());
```

Comment:
I'm getting DejaVu all over again. **It is not valid to used `toLatin1()` on a text that contains translations as `toLatin1()` will nuke any graphemes that are not part of the Latin1 charset which are going to be present in at least *some* of the translations!** If `TConsole::print(...)` requires a `std::string` (or `const char*`) argument then you must use `toUtf8()` instead...

Reply:
Logged as https://github.com/Mudlet/Mudlet/issues/2122